src/components/RoommateGrid/RoommateCard.js

```diff
@@ -0,0 +1,40 @@
+import Typography from "@mui/material/Typography";
+import Card from "@mui/material/Card";
+import CardMedia from "@mui/material/CardMedia";
+import CardContent from "@mui/material/CardContent";
+import CardActions from "@mui/material/CardActions";
+import Button from "@mui/material/Button";
+import Link from "next/link";
+
+export default function RoommateCard({ user }) {
+	// TODO: Default profile picture if we don't have an imageUrl
+	return (
+		<Card variant="outlined" sx={{width: 500}}>
```

Comment:
`width:500` is a bit nonsensical, I just wanted the Card to take up as much width as it could. Let me know if there's a better way.

Reply:
Could also try Fullwidth or set the position and width as this: https://stackoverflow.com/a/44550716 ?